register.go

```diff
@@ -0,0 +1,28 @@
+// Package browser provides an entry point to the browser extension.
+package browser
+
+import (
+	"log"
+	"net/http"
+	_ "net/http/pprof" //nolint:gosec
+	"os"
+
+	"github.com/grafana/xk6-browser/browser"
+
+	k6modules "go.k6.io/k6/js/modules"
+)
+
+func init() {
+	if _, ok := os.LookupEnv("K6_BROWSER_PPROF"); ok {
```

Comment:
I guess we should remember that this won't work with core k6.

Reply:
That's fine tbh.

Out of curiosity, why wouldn't the `init` function run when `k6` imports the module? I thought that all `init` functions had to to run (which is an unwanted side affect sometimes).

Reply:
you import a *package*. 
And we are not going to import *this* package as that will also register the extension as `k6/x/browser` - the init below this one. 

Which is like the whole point of this PR ;)

Reply:
Aha, yep, of course 🙂

Reply:
We use this code in our own instances, so that won't be a problem. Thanks for letting us know, though 👍